client/src/components/ProductItems/ProductItems.module.scss

```diff
@@ -0,0 +1,42 @@
+@import "../../variable.scss";
+
+.ProductItems {
+	* {
+		margin-left: 2%;
+		margin-right: 2%;
```

Comment:
Margins und paddings etc nicht in % angeben.

Reply:
Ok unten hast du bereits Änderung gemacht, habs commited